plugins/system/cpu.go

```diff
@@ -38,13 +54,53 @@ func (s *CPUStats) Gather(acc plugins.Accumulator) error {
 		add(acc, "guest", cts.Guest, tags)
 		add(acc, "guestNice", cts.GuestNice, tags)
 		add(acc, "stolen", cts.Stolen, tags)
+		add(acc, "busy", busy, tags)
+
+		// Add in percentage
+		lastCts := s.lastStats[i]
+		lastBusy, lastTotal := busyAndTotalCpuTime(lastCts)
+		busyDelta := busy - lastBusy
+		totalDelta := total - lastTotal
+
+		if totalDelta < 0 {
+			return fmt.Errorf("Error: current total CPU time is less than previous total CPU time")
+		}
+
+		if totalDelta == 0 {
+			return nil
+		}
+
+		add(acc, "percentageUser", 100*(cts.User-lastCts.User)/totalDelta, tags)
+		add(acc, "percentageSystem", 100*(cts.System-lastCts.System)/totalDelta, tags)
+		add(acc, "percentageIdle", 100*(cts.Idle-lastCts.Idle)/totalDelta, tags)
+		add(acc, "percentageNice", 100*(cts.Nice-lastCts.Nice)/totalDelta, tags)
+		add(acc, "percentageIowait", 100*(cts.Iowait-lastCts.Iowait)/totalDelta, tags)
+		add(acc, "percentageIrq", 100*(cts.Irq-lastCts.Irq)/totalDelta, tags)
+		add(acc, "percentageSoftirq", 100*(cts.Softirq-lastCts.Softirq)/totalDelta, tags)
+		add(acc, "percentageSteal", 100*(cts.Steal-lastCts.Steal)/totalDelta, tags)
+		add(acc, "percentageGuest", 100*(cts.Guest-lastCts.Guest)/totalDelta, tags)
+		add(acc, "percentageGuestNice", 100*(cts.GuestNice-lastCts.GuestNice)/totalDelta, tags)
+		add(acc, "percentageStolen", 100*(cts.Stolen-lastCts.Stolen)/totalDelta, tags)
+
+		add(acc, "percentageBusy", 100*busyDelta/totalDelta, tags)
+
 	}
 
+	s.lastStats = times
+
 	return nil
 }
 
+func busyAndTotalCpuTime(t cpu.CPUTimesStat) (float64, float64) {
```

Comment:
please document this function with a comment stating what the two float64 values are